docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Using Namespaced Broker Resources
+layout: docwithnav
+---
+
+# Cluster-Scoped vs Namespace-Scoped Broker Resources
+
+Service Catalog enables service brokers to be registered in two manners: as a cluster-scoped resource or as a namespace-scoped resource. As a user of service catalog, you might use these approaches to accomplish different goals, like providing a common set of service broker resources to all users or extending your existing role based access control (RBAC) policies to the lifecycle of service broker resources. This document will explain some use cases for namespace-scoped resources and how to accomplish them using Service Catalog.
```

Comment:
Can we get this stuff wrapped at 80 cols?

Reply:
For sure.

Reply:
The mention of extending your existing RBAC policies throws me off a bit here.  Users must have access to the NS to use a specific NS Scoped resource, but that is really the extent of how RBAC comes into play with the NS Scoped resources, right?

Reply:
Yeah, I'll reword this. RBAC in general allows you to prevent people from doing things in the namespace, which includes provisioning. Now the class and plans are in the namespace and get resolved only in the namespace, so if you prohibit someone form provisioning in that namespace, you can prohibit them from provisioning the service all together (which you couldn't really do w/ cluster scope). I'll think of a better way of expressing this. Thanks @jboyd01!